### PR TITLE
[#3800] fix (trino-connector)  EXPLAIN command can not show query optimize details 

### PR DESCRIPTION
**Pull request by diqiu50**
### What changes were proposed in this pull request?

The EXPLAIN command cannot display query optimization details when using the Gravitino Trino connector

### Why are the changes needed?

Fix: #3800

### Does this PR introduce _any_ user-facing change?

NO

### How was this patch tested?

IT

### CODE REVIEW

**Comment by diqiu50** [comment]
I refactored the stored procedure to move the implementation code to a specific `XXXStoredProcedure` class.

**Comment by yuqi1129** [comment]
@diqiu50 
Can you provide a brief description of the code changes you have made?

**Comment by diqiu50** [comment]
The "Explain" command's details depend on TableHandle::toString. I override the toString method in TableHandle and pass the call to the internal table handle.

> @diqiu50 Can you provide a brief description of the code changes you have made?

**Comment by yuqi1129** [comment]
There is a typo by the way: 

```
    <artifactId>trino-gravitino</artifactId>
    <packaging>trino-plugin</packaging>
    <description>Trino - Graviton Connector</description>
```

File: trino-connector/src/main/java/com/datastrato/gravitino/trino/connector/catalog/CatalogConnectorManager.java [hunk 1]
```diff
@@ -258,151 +254,6 @@ public String getTrinoCatalogName(GravitinoCatalog catalog) {
     return getTrinoCatalogName(catalog.getMetalake(), catalog.getName());
   }
 
-  public void createCatalog(
```
[the remark this answers is not in the document]
Comment (diqiu50) [reply trino-connector/src/main/java/com/datastrato/gravitino/trino/connector/catalog/CatalogConnectorManager.java:261]: I think the logic should belong to a specific stored procedure. The CatalogConnectorManager is used to manage catalog connectors.

**Review by yuqi1129 — APPROVED** [review]
LGTM

**Pull request merged**